src/Core/Configure.php

```diff
@@ -138,6 +139,32 @@ public static function check($var)
     }
 
     /**
+     * Used to get information stored in Configure. It's not
+     * possible to store `null` values in Configure.
+     *
+     * Acts as a wrapper around Configure::read() and Configure::check().
+     * The configure key/value pair fetched via this method is expected to exist. 
+     * In case it does not an exception will be thrown.
+     *
+     * Usage:
+     * ```
+     * Configure::readOrFail('Name'); will return all values for Name
+     * Configure::readOrFail('Name.key'); will return only the value of Configure::Name[key]
+     * ```
+     *
+     * @param string $var Variable to obtain. Use '.' to access array elements.
+     * @return mixed Value stored in configure.
+     * @throws \RuntimeException if the requested configuration is not set.
+     * @link http://book.cakephp.org/3.0/en/development/configuration.html#reading-configuration-data
+     */
+    public static function readOrFail($var) {
+        if (static::check($var) === false) {
+            throw new RuntimeException(sprintf('Expected "%s" configuration.', $var));
```

Comment:
`'Expected configuration key "%s" not found.'`  might be more easily understandable to users.